include/nvector/nvector_trilinos.h

```diff
@@ -98,7 +98,7 @@ SUNDIALS_EXPORT N_Vector N_VCloneEmpty_Trilinos(N_Vector w);
 SUNDIALS_EXPORT N_Vector N_VClone_Trilinos(N_Vector w);
 SUNDIALS_EXPORT void N_VDestroy_Trilinos(N_Vector v);
 SUNDIALS_EXPORT void N_VSpace_Trilinos(N_Vector v, sunindextype *lrw, sunindextype *liw);
-SUNDIALS_EXPORT void *N_VGetCommunicator_Trilinos(N_Vector v);
+SUNDIALS_EXPORT MPI_Comm N_VGetCommunicator_Trilinos(N_Vector v);
```

Comment:
Should this be `SUNComm`?
```suggestion
SUNDIALS_EXPORT SUNComm N_VGetCommunicator_Trilinos(N_Vector v);
```